ibis/backends/postgres/compiler.py

```diff
@@ -330,7 +330,10 @@ def visit_ToJSONArray(self, op, *, arg):
         )
 
     def visit_Map(self, op, *, keys, values):
-        return self.f.map(self.f.array(*keys), self.f.array(*values))
+        # map(["a", "b"], NULL) results in {"a": NULL, "b": NULL} in regular postgres,
```

Comment:
Wow, that's ... weird.

Reply:
I didn't look more into this, but I'm wondering if this allows you to use Maps as a HashSet, which actually is pretty cool. Maybe we even want to change ibis semantics to allow this?? Idk haven't thought how that would interact with other backends

Reply:
interesting idea, but let's leave it for later :)